### DISCUSSION TITLE
[Bug Report] I just added my library and no scenes are showing up

### DISCUSSION THREAD

**Discussion by niveaboy1** [stashapp/stash#4258] [Q&A] [answered]
I just installed and set up stash, added my library and can't see any scenes showing up. I'm using version 0.23.1 on firefox on windows os. 
Please help how I can fix this issue. Thanks!
![stash 2](https://github.com/stashapp/stash/assets/64467249/4d79afb3-dcac-454c-a276-64ae29af1836)
![Stash](https://github.com/stashapp/stash/assets/64467249/91909902-03de-496e-9f95-a9cb5e333ad4)

**Comment by QxxxGit** [comment, marked as answer]
Have you ran the Scan task on the Tasks page? This will dig through your library, generate necessary files, and add information to your database.

This doesn't seem to be a bug report, so I recommend joining the [Discord](https://discord.gg/2TsNFKt) and asking for help there. You'll receive assistance pretty quickly from the community.
